src/tools/Circle.ts

```diff
@@ -3,11 +3,12 @@ import Konva from 'konva'
 import { CanvasElement } from '@/types/Canvas'
 import uuid from 'uuid'
 import throttle from 'lodash.throttle'
+import CircleCreator from '@/tools/shapes/CircleCreator'
 
 export default class Circle implements Tool {
   private circle: Konva.Circle;
-  private line: Konva.Line;
-  private stroke: number[][];
+  private line?: Konva.Line;
```

Comment:
I am surprised it hasn't complained when using `;` can we remove them?

Reply:
Will remove